clients/admin-ui/src/pages/management/domains.tsx

```diff
@@ -91,38 +155,41 @@ const CORSConfigurationPage: NextPage = () => {
   };
 
   return (
-    <Layout title="Manage domains">
-      <Box data-testid="cors-configuration">
+    <Layout title="Domains">
+      <Box data-testid="management-domains">
         <Heading marginBottom={4} fontSize="2xl">
-          Manage domains
+          Domains
         </Heading>
         <Box maxWidth="600px">
-          <Text marginBottom={2} fontSize="md">
-            Manage domains for your organization
-          </Text>
           <Text mb={10} fontSize="sm">
-            You must add domains associated with your organization to Fides to
-            ensure features such as consent function correctly. For more
-            information on managing domains on Fides, click here{" "}
-            <DocsLink href="https://fid.es/cors-configuration">
-              docs.ethyca.com
+            For Fides to work on your website(s), each of your domains must be
+            listed below. You can add and remove domains at any time up to the
+            quantity included in your license. For more information on managing
+            domains{" "}
```

Comment:
I like the user-friendly wording here with the CORS details in the tooltip 👌

Reply:
👍